CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 -
 
 ### Changed
--
+- **capital-framework:** [PATCH] Standardized all the usage docs for v4
```

Comment:
If you want every individual component to get a patch bump, switch this to `all components`. Otherwise it'll patch bump [capital-framework](https://www.npmjs.com/package/capital-framework) and not bump any of the components. This is intentional because sometimes we just update trivial stuff like the main README and there's no reason to bump the components in that scenario.

It looks like this PR is updating the docs for the individual components so `all components` is more appropriate because we want every component to get tagged and bumped with the new docs. Instead of putting `all components` you could also list every component on its own line but the `all components` option was added as a convenience to save tedium.

We should revisit CF's continuous integration scripts now that we've been using them for 1+ years. There's definitely ways to make things less confusing.